pylint/test/functional/mutable_sequence_modified_in_loop.py

```diff
@@ -0,0 +1,75 @@
+
+
+""" The mutable sequence was modified inside the loop """
+
+# pylint:disable=print-statement, missing-docstring, redefined-outer-name, invalid-name, using-constant-test, no-member
+
+from __future__ import print_function
+
+
+class MyList(object):
+
+    def __init__(self):
+        self.my_list = range(1, 6)
+
+    def __iter__(self):
+        return iter(self.my_list)
+
+    def remove(self, item):
+        pass
+
+    def delete(self, item):
+        pass
+
+    def pop(self, item):
+        pass
+
+
+my_list = MyList()
+for item in my_list:
+    print(item)
```

Comment:
Let's remove this ``print``, we don't need it since we are already using ``item``. This will result in some pragma disables being removed and generally in simpler test code.